packages/app/src/systems/DApp/machines/transactionRequestMachine.tsx

```diff
@@ -92,6 +92,7 @@ export const transactionRequestMachine = createMachine(
     },
     states: {
       idle: {
+        tags: ['loading'],
```

Comment:
The loading state should be only on the icon using the tag it will put the entire page on loading state, no?

Reply:
I really don't know why this is there 😅